openupgrade_scripts/scripts/stock_account/14.0.1.1/noupdate_changes.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version='1.0' encoding='utf-8'?>
 <odoo>
-  <!-- <record id="stock_valuation_layer_company_rule" model="ir.rule">
-    <field name="global"/>
-  </record> -->
```

Comment:
please don't delete this. we autogenerate this kind of data, and we don't want to think again if this needs to be commented or not when it reappears if we run the script again.